index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  equals: 'equals',
+  concat: 'concat',
+  empty: 'empty',
+  map: 'map',
+  ap: 'ap',
+  of: 'of',
+  reduce: 'reduce',
+  sequence: 'sequence',
+  chain: 'chain',
+  extend: 'extend',
+  extract: 'extract'
+}
```

Comment:
Did we not want to go the full symbols route or at least try it so we can see if it works or not?